elasticdl/python/tests/model_handler_test.py

```diff
@@ -152,21 +164,31 @@ def setUp(self):
             checkpoint_dir="",
         )
 
-    def _save_model(self, is_subclass):
-        prefix = "custom_model/" if is_subclass else ""
-        ckpt_dir = self.model_handler._checkpoint_dir
-        checkpoint_saver = CheckpointSaver(ckpt_dir, 0, 0, False)
+    def _mock_model_parameters(self, model):
         params = Parameters()
-        params.non_embedding_params[
-            prefix + "embedding/embeddings:0"
-        ] = tf.Variable(tf.ones([EMBEDDING_INPUT_DIM, 2]), dtype=tf.float32)
-        params.non_embedding_params[prefix + "dense/kernel:0"] = tf.Variable(
-            [[1.0], [1.0]]
-        )
-        params.non_embedding_params[prefix + "dense/bias:0"] = tf.Variable(
-            [1.0]
-        )
+        for weight in model.trainable_variables:
+            if "embedding" in weight.name:
+                embedding_table = EmbeddingTable(
+                    name=weight.name,
+                    dim=weight.shape[1],
+                    initializer="RandomUniform",
+                )
+                embedding_table.set(
+                    np.arange(weight.shape[0]), np.ones(weight.shape)
+                )
+                params.embedding_params[weight.name] = embedding_table
+            else:
+                params.non_embedding_params[weight.name] = tf.ones(
+                    weight.shape
+                )
         params.version = 100
+        return params
+
+    def _save_model(self, model):
```

Comment:
_fill_model_with_mock_weights_and_save_checkpoint

Reply:
_mock_model_weights_and_save_checkpoint ?

Reply:
`_mock_model_weights_and_save_checkpoint` is good.